opendevin/runtime/browser/browser_env.py

```diff
@@ -32,12 +33,18 @@ def __init__(self):
         self.process = multiprocessing.Process(
             target=self.browser_process,
         )
+        if is_async:
+            threading.Thread(target=self.init_browser).start()
+        else:
+            self.init_browser()
+        atexit.register(self.close)
+
+    def init_browser(self):
```

Comment:
Do we want to impelement the lazy luanching or async launching?

Reply:
I would argue that we could add that to a different PR if necessary, this is already an improvement.

Reply:
Looks like this PR doesn't only make it optional (lazy launching), but also make the launching itself async. Two improvements.